gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -385,7 +456,7 @@ def _decrement_device_interface_count(self, device):
         self._update_network_function_device_db(device, device['status'])
 
     def _get_orchestration_driver(self, service_vendor):
-        return self.drivers[service_vendor.lower()]
+        return orchestration_driver.OrchestrationDriver(self.config)
```

Comment:
Every time this method is invoked a new object is created. Can we keep it instantiated in self and return the same in this method. What is the difference between these approaches.